cms/toolbar/toolbar.py

```diff
@@ -97,7 +98,7 @@ def init_toolbar(self, request):
         self.request = request
         self.is_staff = self.request.user.is_staff
         self.edit_mode = self.is_staff and self.request.session.get('cms_edit', False)
-        self.build_mode = self.is_staff and self.request.session.get('cms_build', False)
+        self.build_mode = self.is_staff and request.GET.get(get_cms_setting('CMS_TOOLBAR_URL__BUILD')) != None
```

Comment:
the usecase is sharing a link to another cms user (will go into correct mode). it is documented here http://docs.django-cms.org/en/latest/reference/api_references.html?highlight=build_mode#module-cms.toolbar.toolbar but it's not respected in any way at the moment